modules/probe.sh

```diff
@@ -94,6 +94,49 @@ probe_debian_arch_name() {
   echo "${deb_arch}"
 }
 
+# For x86 distros like e.g. Ubuntu try to find grubx64.efi
+# file and return it's path otherwise return empty path
+#
+# No input parameters and these work on the assumption that boot and root parts
+# are mounted at work/boot
+find_grub_efi_path() {
+  echo "$(sudo find work/boot -name grubx64.efi)"
```

Comment:
I'm nitpicking a lot now, but my point was that `echo` is entirely unnecessary, you can just call `find` directly. :-)

Reply:
I'm using this method in 2 places that is the reason why I put it to function ;)

Reply:
You can still do that. This is what I mean:

```bash
find_grub_efi_path() {
  sudo find work/boot -name grubx64.efi
}
```

Reply:
ok I get it now. Will update ;)

Reply:
Updated